src/api/archive_unstable_storageDiff.md

```diff
@@ -0,0 +1,98 @@
+
+  - `childTrieKey`: String containing the hexadecimal-encoded key of the child trie of the "default" namespace. This field is only present when the `trieType` field is set to `childTrie`.
+
+**Return value**: A JSON object.
+
+The JSON object returned by this function has the following format:
+
+```json
+{
+    "result": [
+        {
+            "differences": [
+                {
+                    "key": "0x...",
+                    "value": "0x...",
+                    "hash": "0x...",
+                    "type": "added" | "modified" | "deleted",
+                },
+            ],
+
+            "trieType": "mainTrie" | "childTrie",
+            "childTrieKey": "0x...",
+        }
+    ],
+}
+```
+
+The `result` field contains an array of objects, each containing a JSON object:
+
+The `differences` field is an array of objects describing storage differences. Each element contains the following fields:
+
+- `key`: String containing the hexadecimal-encoded key of the storage entry. If the key prefix was provided in the `items` parameter, the `key` field is guaranteed to start with one of the key prefixes provided.
+
+- `value`: String containing the hexadecimal-encoded value of the storage entry. This field is present when prefixes are not provided or when the `type` field in the `items` parameter is set to `value`.
+
+- `hash`: String containing the hexadecimal-encoded hash of the storage entry. This field is only present when the `type` field in the `items` parameter is set to `hash`.
+
+- `type`: String indicating the type of the storage difference. The possible values are:
+  - `added`: The storage entry was added.
+  - `modified`: The storage entry was modified.
+  - `deleted`: The storage entry was deleted.
+
+The `trieType` field is a string indicating the trie type for which the storage difference was calculated. The possible values are `mainTrie` and `childTrie`. These have the same meaning as in the `items` parameter.
+
+The `childTrieKey` field is a string containing the hexadecimal-encoded key of the child trie of the "default" namespace. This field is only present when the `trieType` field is set to `childTrie`.
+
+## Overview
+
+This function calculates the storage difference between two blocks. The storage difference is calculated by comparing the storage of the `previousHash` block with the storage of the `hash` block. If the `previousHash` parameter is not provided, the storage difference is calculated between the parent of the `hash` block and the `hash` block.
+
+The JSON-RPC server is encouraged to accept at least one `archive_unstable_storageDiff` subscription per JSON-RPC client. Trying to open more might lead to a JSON-RPC error when calling `archive_unstable_storageDiff`. The JSON-RPC server must return an error code if the server is overloaded and cannot accept new subscriptions.
```

Comment:
Subscription? If I read the description properly this just makes a method call with the storageDiff as return value

Reply:
Opps, that was added before, thanks for catching this 🙏